core/videolib/convertor.py

```diff
@@ -43,22 +45,20 @@ def get_ts_start_time(file_path: str) -> float:
     float
         The start time of the .ts file located at file_path
     """
-
     parse_png_to_mpeg2ts_stream(file_path)
     return float(os.path.split(file_path)[-1])
 
 
 def parse_png_to_mpeg2ts_stream(file_path: str) -> None:
-    """Strip mpeg2-ts from downloaded file
+    """Strip the mpeg2-ts data from a file.
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/manual/excalibur.krv/m3u8-dl/pullRequest?prid=6274429)